l2geth/rollup/sync_service.go

```diff
@@ -856,10 +857,14 @@ func (s *SyncService) applyTransaction(tx *types.Transaction, txSetProof *types.
 			log.Info("message.UnPackData interrupt", "error", err)
 		}
 
-		sccAddress := s.RollupGpo.SCCAddress()
-		var zeroAddr common.Address
-		if sccAddress == zeroAddr {
-			log.Crit("RollupGpo get sccAddress", "error", fmt.Errorf("sccAddr is zeroAddr"))
+		var zeroAddr, sccAddress common.Address
+		for {
+			sccAddress = s.RollupGpo.SCCAddress()
+			if sccAddress != zeroAddr {
+				break
+			}
+			log.Error("RollupGpo get sccAddress", "error", fmt.Errorf("sccAddr is zeroAddr"))
+			time.Sleep(retryInterval)
```

Comment:
Add max retry and exit this process if sccAddress  always is zero address

Reply:
I was going to let it keep retrying, because there's no point in jumping out, and if we keep retrying, all we have to do is wait or kill the process